packages/gce-disk-expand/src/expandfs-lib.sh

```diff
@@ -78,7 +84,14 @@ sgdisk_fix_gpt() {
   local label=$(sgdisk_get_label "$disk")
   [ "$label" != "gpt" ] && return
 
+  # Add sleeps to allow this operation to fully complete. On some systems, other
+  # operations such as systemd-fsck tasks can collide and fail.
+  udevadm settle
+  sleep 1
```

Comment:
Please add a TODO to remove/replace these sleeps with better logic when we figure out a better alternative.

Reply:
Done